crates/driver/src/domain/eth/gas.rs

```diff
@@ -2,6 +2,7 @@ use {
     super::{Ether, U256},
     bigdecimal::Zero,
     num::zero,
+    number::U256 as NumberU256,
```

Comment:
Not idea to have the two types `primite_type::U256` and `number::U256`. ideally everything would be moved to `number::U256`.